src/index.tsx

```diff
@@ -24,10 +24,24 @@ import CurrentTime from './CurrentTime'
 import Duration from './Duration'
 import VolumeBar from './VolumeBar'
 import { RHAP_UI, MAIN_LAYOUT, AUDIO_PRELOAD_ATTRIBUTE, TIME_FORMAT } from './constants'
-import { throttle, getMainLayoutClassName } from './utils'
+import { throttle, getMainLayoutClassName, getDisplayTimeBySeconds } from './utils'
 
 type CustomUIModule = RHAP_UI | ReactElement
 type CustomUIModules = Array<CustomUIModule>
+type OnSeek = (audio: HTMLAudioElement, time: number) => Promise<void>
+
+interface AudioChunk {
+  url: string
+  duration: number
+  data?: AudioBuffer
+}
+
+interface MSEPropsObject {
+  onSeek?: OnSeek
+  onEcrypted?: (e: unknown) => void
+  srcDuration?: number
+  playlist?: AudioChunk[]
```

Comment:
The object itself is optional, but some/all of the fields seems required, right?

Reply:
Right, `playlist` was a leftover from me testing importing the chunk logic that we will do later.
The rest only `onEcrypted` is not required.